_config.yml

```diff
@@ -27,7 +27,7 @@ descriptionlong: >- # this means to ignore newlines until "baseurl:"
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://plait-lab.org" # the base hostname & protocol for your site, e.g. http://example.com
 twitter_username: PLAITlab
-github_username:  plait-lab
+github_username: plait-lab
```

Comment:
These are just little whitespace corrections introduced by my editor due to `formatOnSave` being set to true. Happy to back them out, but looks like it's just removing erroneous whitespace 😄